exampleservice/CustomerService/Handler/CustomerHandlerBase.cs

```diff
@@ -0,0 +1,20 @@
+﻿using exampleservice.Framework.BaseFramework;
+using System;
+using System.Threading.Tasks;
+
+namespace exampleservice.CustomerService.Handler
+{
+    internal abstract class CustomerHandlerBase<T> where T : CustomerCommandBase
```

Comment:
Just for the record: I actively decided against a base-class because I think that the amount "saved" LOC isn't worth the 
loss in readability. Especially because we do not need/use the polymorphism at any point yet. What was your intention on adding this (out of curiosity)?

Reply:
did some refactoring and the readability is different now anyways so whatever ^^